packages/smooth_app/lib/pages/guides/guide/guide_open_prices.dart

```diff
@@ -0,0 +1,258 @@
+
+    return GuidesPage(
+      pageName: 'OpenPrices',
+      header: const _OpenPricesHeader(),
+      body: const <Widget>[
+        _OpenPricesSection1(),
+        _OpenPricesSection2(),
+        _OpenPricesSection3(),
+        _OpenPricesSection4(),
+        _OpenPricesSection5(),
+      ],
+      footer: SliverToBoxAdapter(
+        child: GuidesFooter(
+          shareUrl: appLocalizations.guide_open_prices_share_link,
+        ),
+      ),
+    );
+  }
+}
+
+class _OpenPricesHeader extends StatelessWidget {
+  const _OpenPricesHeader();
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return GuidesHeader(
+      title: appLocalizations.guide_open_prices_title,
+      illustration: const _OpenPricesHeaderIllustration(),
+    );
+  }
+}
+
+class _OpenPricesHeaderIllustration extends StatelessWidget {
+  const _OpenPricesHeaderIllustration();
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: AlignmentDirectional.centerEnd,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: <Widget>[
+          Expanded(
+            flex: 32,
+            child: SvgPicture.asset(
+              'assets/guides/open_prices/open_prices_logo.svg',
+              width: 120.0,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _OpenPricesSection1 extends StatelessWidget {
+  const _OpenPricesSection1();
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return GuidesParagraph(
+      title: appLocalizations.guide_open_prices_what_is_open_prices_title,
+      content: <Widget>[
+        GuidesText(
+          text:
+              appLocalizations.guide_open_prices_what_is_open_prices_paragraph1,
+        ),
+        GuidesText(
+          text:
+              appLocalizations.guide_open_prices_what_is_open_prices_paragraph2,
+        ),
+      ],
+    );
+  }
+}
+
+class _OpenPricesSection2 extends StatelessWidget {
+  const _OpenPricesSection2();
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return GuidesParagraph(
+      title: appLocalizations.guide_open_prices_how_title,
+      content: <Widget>[
+        GuidesText(text: appLocalizations.guide_open_prices_how_paragraph1),
+        GuidesTitleContainer(
+          icon: const Icon(Icons.sell_outlined, color: Colors.white),
+          title: appLocalizations.guide_open_prices_how_arg1_title,
+          child: Column(
+            spacing: MEDIUM_SPACE,
+            children: <Widget>[
+              ClipRRect(
+                borderRadius: ROUNDED_BORDER_RADIUS,
+                child: Image.network(
+                  'https://prices.openfoodfacts.org/img/0029/nCWeCVnpQJ.400.webp',
```

Comment:
Are we sure that this link we last forever?
Shouldn't we cache it?

Reply:
@teolemon what do you think? You provided me with this link

Reply:
It's showcased on https://prices.openfoodfacts.org/contribute, so I expect a level of stability @raphodn @TTalex ?

Reply:
1. We should store the image (offline mode)
2. A sharp (= without blur), better-cropped image is expected